_pages/bea/bea19.md

```diff
@@ -98,32 +96,119 @@ Gold Sponsors
+Submissions failing to meet required styles, including paper size, margin width, and font size restrictions, will be rejected without review.
+
+Limitations
+: Authors are required to discuss the limitations of their work in a dedicated section titled "Limitations".
+This section should be included at the end of the paper, before the references, and it will not count toward the page limit.
+This includes both, long and short papers.
+Note, prior to the December 2023 cycle, this was optional.
+
+Ethics Policy
+: Authors are required to honour the ethical code set out in the [ACL Code of Ethics](https://www.aclweb.org/portal/content/acl-code-ethics).
+The consideration of the ethical impact of our research, use of data, and potential applications of our work has always been an important consideration, and as artificial intelligence is becoming more mainstream, these issues are increasingly pertinent.
+We ask that all authors read the code, and ensure that their work is conformant to this code.
+Authors are encouraged to devote a section of their paper to concerns about the ethical impact of the work and to a discussion of broader impacts of the work, which will be taken into account in the review process.
+This discussion may extend into a 5th page (short papers) or 9th page (long papers).
+
+Anonymity
+: Given the blind review process, it is essential to ensure that papers remain anonymous.
+Authors should avoid self-references that disclose their identity (e.g., “We previously showed (Smith, 1991)”), opting instead for citations like “Smith previously showed (Smith, 1991)”.
+
+Conflicts of Interest
+: Authors are required to mark potential reviewers who have co-authored the paper, belong to the same research group or institution, or have had prior exposure to the paper, ensuring transparency in the review process.
+
+Double Submissions
+: We adhere to the official ACL double-submission policy.
+If papers are submitted to both BEA and another conference or workshop, authors must specify the other event on the title page (as a footnote on the abstract).
+Additionally, the title page should state that if the paper is accepted for presentation at BEA, it will be withdrawn from other conferences and workshops.
+
+Republications
+: Previously published papers will not be accepted.
+
+### Presentation Guidelines
+
+All accepted papers must be presented at the workshop to appear in the proceedings.
+The workshop will include both in-person and virtual presentation options.
+**At least one author of each accepted paper must register for the conference by the early registration deadline.**
+
+Long and short papers will be presented orally or as posters as determined by the workshop organizers.
+While short papers will be distinguished from long papers in the proceedings, there will be no distinction in the proceedings between papers presented orally and papers presented as posters.
```

Comment:
I copied this from the NAACL conference. Might be interesting to mention explicitly. I've had these questions from co-authors outside of the ACL/BEA community.